### PR TITLE
Implement languages configuration page

### PR DESCRIPTION
**Pull request by cychiuae**
ref #4289

### CODE REVIEW

File: portal/src/graphql/portal/LanguagesConfigurationScreen.tsx [hunk 1]
```diff
@@ -80,16 +89,18 @@ const SelectPrimaryLanguageSection: React.VFC<SelectPrimaryLanguageWidgetProps>
       onChangePrimaryLanguage,
     } = props;
 
+    const { getLanguageDisplayText } = useContext(PageContext);
+
     const [searchValue, setSearchValue] = useState("");
     const dropdownOptions: IDropdownOption[] = useMemo(() => {
       const filteredLanguages = availableLanguages.filter((lang) =>
         lang.toLowerCase().includes(searchValue.toLowerCase())
       );
```
Comment (tung2744) [inline portal/src/graphql/portal/LanguagesConfigurationScreen.tsx]: Should the user be able to search by display name?

Comment (tung2744) [reply portal/src/graphql/portal/LanguagesConfigurationScreen.tsx]: Saw it fixed in later commit

File: portal/src/graphql/portal/LocalizationConfigurationScreen.tsx [hunk 2]
```diff
@@ -900,22 +820,23 @@ const LocalizationConfigurationScreen: React.VFC =
...
-    const config = useAppConfigForm({
-      appID,
-      constructFormState,
-      constructConfig,
-    });
+    // const config = useAppConfigForm({
+    //   appID,
+    //   constructFormState,
+    //   constructConfig,
+    // });
```
Comment (tung2744) [inline portal/src/graphql/portal/LocalizationConfigurationScreen.tsx:827]: It seems this comment can be removed

**Comment by tung2744** [comment]
Screenshot 2024-06-14 at 1 36 29 PM

The cross button here has no response. And from figma it should be a down arrow. Maybe we should props to SearchableDropdownSearchBox to control the right button. (Disable the button, and use another icon)

**Comment by cychiuae** [comment]
> The cross button here has no response. And from figma it should be a down arrow. Maybe we should add props to SearchableDropdownSearchBox to control the right button. (Disable the button, and use another icon)

Oh didn't notice it

File: portal/src/graphql/portal/LocalizationConfigurationScreen.tsx (hunk 2 above)
Comment (cychiuae) [reply portal/src/graphql/portal/LocalizationConfigurationScreen.tsx]: I fixed up in [0df1192](https://github.com/authgear/authgear-server/pull/4331/commits/0df1192f2f05a60926d02ba490dbc74e699d49e2)

**Review by tung2744 — APPROVED** [review]

**Pull request merged**